### DISCUSSION TITLE
Searchbox with Elastic App Search

### DISCUSSION THREAD

**Discussion by nageshavusingi** [appbaseio/reactivesearch#1805] [Q&A] [answered]
Team, Can some one point to an example using Searchbox (or Reactive search components) with Elastic App Search hosted on elastic.co cloud?

I have access to Engine (not index name), search and private keys and end point URL. 

Couldn't find examples to configure SearchBase (or ReactiveBase in case of Reactive search) with elastic cloud credentials. (engine name, search key and end point URL)

**Comment by siddharthlatest** [comment, marked as answer]
@nageshavusingi ReactiveSearch/Searchbox libraries currently don't support directly using Elastic App Search service. They're primarily meant to be used with Elasticsearch.
